### DISCUSSION TITLE
How to handle errors of smiles conversion.

### DISCUSSION THREAD

**Discussion by khoroshyy** [rdkit/rdkit#4902] [Q&A] [answered]
Hi,
I am converting lists of smiles, but they sometimes have errors in syntax and then the function breaks.
What is the proper way to handle errors of 
Chem.MolFromSmiles(smiles_list[i])
function?
Thanks.
Petro.

**Comment by greglandrum** [comment, marked as answer]
@khoroshyy : this is a question which has been asked repeatedly here and in other places. Please try doing a web search for the error message you get. 

If you don't find an answer there which helps you, then include the SMILES you are using and the exact error message you got from the RDKit in your question.

**Reply by khoroshyy** [reply]
I solved it.
The problem was that I have had numbers instead of strings in list.
So I get none with "44" but code breaks if I had 44 integer.
